src/devices_models/device_constructors/branch_constructor.jl

```diff
@@ -1,33 +1,78 @@
 # These 3 methods are defined on concrete formulations of the branches to avoid ambiguity
 construct_device!(
+    ::ArgumentConstructStage,
```

Comment:
The container should be passed first since it is the mutated argument?

Reply:
Resolved in 169069a.